service/src/main/java/eu/europa/ec/fisheries/uvms/activity/service/FishingTripCache.java

```diff
@@ -7,7 +7,7 @@
 import java.util.HashMap;
 import java.util.Map;
 
-public class FaReportMessageFishingTrips {
+public class FishingTripCache {
```

Comment:
Do we support reuse of an existing trip if we got it in a previous flux document?

Reply:
No, so this is still an issue. We need tests for that.